.changes/next-release/feature-28e0ea34-b749-4750-8ee2-bd8c92be78df.json

```diff
@@ -0,0 +1,4 @@
+{
+  "type" : "feature",
+  "description" : "Promoting experimental proxy support to be the default behavior. Fixes #685."
```

Comment:
We never actually made a changelog about it, so you could just say Added proxy support, or Respects IDE's proxy settings